css/templates/travel/page.css

```diff
@@ -0,0 +1,42 @@
+/**
+  * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *      http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS-IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+
+/*! Bassplate | MIT License | http://github.com/basscss/bassplate */
+@import 'normalize.css';
```

Comment:
Most of this should exist in core, can you check some other templates and how they have done this

Reply:
I believe our reasoning behind this was include only some imports from core to keep the css size down.